samples/notebooks/openqasm.ipynb

```diff
@@ -0,0 +1,359 @@
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from qsharp.code import bell\n",
+        "\n",
+        "qsharp.circuit(bell)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "5865ea0c",
+      "metadata": {},
+      "source": [
+        "In notebooks, we can do a bit better leveraging the circuit widget:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "c3fc87ae",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from qsharp.code import bell\n",
+        "from qsharp_widgets import Circuit\n",
+        "\n",
+        "Circuit(qsharp.circuit(bell))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "0e1a29c8",
+      "metadata": {},
+      "source": [
+        "And finally when getting ready to submit to hardware, we can compile the program to QIR:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "ec6db1e5",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from qsharp.code import bell\n",
+        "\n",
+        "print(qsharp.compile(bell))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "759f6a5e",
+      "metadata": {},
+      "source": [
+        "We can also define input for the compiled OpenQASM code so that we can parameterize input with imported callables:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "bd179f1a",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from qsharp import init, TargetProfile\n",
+        "from qsharp.openqasm import import_openqasm\n",
+        "\n",
+        "source = \"\"\"\n",
+        "include \"stdgates.inc\";\n",
+        "input float theta;\n",
+        "bit[2] c;\n",
+        "qubit[2] q;\n",
+        "rx(theta) q[0];\n",
+        "rx(-theta) q[1];\n",
+        "c = measure q;\n",
+        "\"\"\"\n",
+        "\n",
+        "init(target_profile=TargetProfile.Base)\n",
+        "import_openqasm(source, name=\"parameterized_program\")\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "5b7a97d2",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from qsharp.code import parameterized_program\n",
+        "from qsharp.openqasm import compile\n",
+        "\n",
+        "print(compile(parameterized_program, 1.57))"
+      ]
+    }
+  ],
+  "metadata": {
+    "kernelspec": {
+      "display_name": ".venv",
+      "language": "python",
+      "name": "python3"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.13.3"
+    }
+  },
+  "nbformat": 4,
+  "nbformat_minor": 5
+}
```

Comment:
Nice! I really like the clarity and simplicity of this notebook. It flows really well.